packages/documentation/package.json

```diff
@@ -18,14 +18,20 @@
     "start": "start-storybook -p 9200 --quiet",
     "start:headless": "start-storybook -p 9300 --quiet --no-open",
     "build": "build-storybook --quiet",
-    "test": "start-server-and-test start:headless 9300 'percy exec -- cypress run --record false'"
+    "test": "start-server-and-test start:headless 9300 'percy exec -- cypress run --record false'",
+    "cypress:open": "cypress open --config baseUrl=http://localhost:9200",
+    "cypress:run": "cypress run",
+    "cypress:dev": "start-server-and-test start http://localhost:9200 cypress:open"
   },
   "dependencies": {
     "@swisspost/design-system-components": "workspace:1.0.4",
+    "@swisspost/design-system-components-react": "workspace:0.0.1",
     "@swisspost/design-system-styles": "workspace:5.1.3",
+    "html-react-parser": "3.0.4",
+    "js-beautify": "1.14.7",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "html-react-parser": "3.0.4"
```

Comment:
The `html-react-parse` package is used within the `alert.stories.tsx`.
Try to replace it there as well.

Reply:
The package is still there, it was just moved because of the automatic alphabetization.